app/utils/misc.ts

```diff
@@ -1,4 +1,6 @@
+import { ClassNameValue } from 'node_modules/tailwind-merge/dist/lib/tw-join.js'
 import React from 'react'
+import { twMerge, twJoin } from 'tailwind-merge'
```

Comment:
I think instead of twJoin it should be using clsx, the docs of tailwind-merge says that twJoin is similar expect it doesn’t support object syntax that clsx supports, and if it already use clsx I see no reason to remove a possible feature.

Reply:
I have mixed feelings about this. Their reasoning for no object syntax is pretty good, but I really hate abusing `&&` like that. I'm going to think about this more. I don't like having two ways to do the same thing 😕

Reply:
More than two ways people should just use consolidate, maybe it be a separate package that combines both if you don’t want to have clsx and tailwind-merge installed together.